### PR TITLE
vlang: 2022.20 -> 2022.43

### PR DESCRIPTION
**Pull request by rilysh**
###### Description of changes
Version bump from 2022.20 to 2022.43

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by 9glenda — APPROVED** [review]

**Comment by tengkuizdihar** [comment]
2 days ago, `0.3.2` is released which is newer than this. Shouldn't we just use that one?

File: pkgs/development/compilers/vlang/default.nix [hunk 1]
```diff
@@ -2,29 +2,29 @@
 
 stdenv.mkDerivation rec {
   pname = "vlang";
-  version = "weekly.2022.20";
+  version = "weekly.2022.43";
```
Comment (tengkuizdihar) [inline pkgs/development/compilers/vlang/default.nix:5]: 2 days ago, 0.3.2 is released which is newer than this. Shouldn't we just use that one?

**Comment by tengkuizdihar** [comment]
I've also tried to compile this derivation locally but can't get it working, am I doing something wrong? I'm basically doing this

```nix
let 
  pkgs = import (builtins.fetchTarball "https://github.com/nixos/nixpkgs/archive/d01cb18be494e3d860fcfe6be4ad63614360333c.tar.gz") {};
in
pkgs.stdenv.mkDerivation {
    name = "littlemllibrary";
    buildInputs = [
        (pkgs.callPackage ./nix/vlang.nix {})
    ];
}
```

and vlang.nix is the file of this PR.

**Comment by tokopedihar** [comment]
hello, any update on this one?

**Comment by knarkzel** [comment]
Can we bump this to 0.3.3?

**Comment by Artturin** [comment]
https://github.com/NixOS/nixpkgs/pull/231959

**Pull request closed without merging**